locale/fr/LC_MESSAGES/django.po

```diff
@@ -4983,7 +4983,7 @@ msgstr "Divers"
 
 #: zerver/lib/integrations.py:57
 msgid "Monitoring"
-msgstr "Suivi "
+msgstr "Suivi"
```

Comment:
Fixed this in Transifex at https://app.transifex.com/zulip/zulip/translate/#fr/djangopo/437775282?q=text%3Amonitoring.